framework/platform/bpl/linux/prplmesh_platform_db

```diff
@@ -1,4 +1,4 @@
 management_mode=Multi-AP-Controller-and-Agent
 operating_mode=Gateway
-stop_on_failure_attempts=0
+stop_on_failure_attempts=1
```

Comment:
I'm not convinced... We also want a test_flows for agent certification, and that will have to do dev_reset_default...

Reply:
we don't decrement stop_on_failure_attempts when backhaul disconnects so setting stop_on_failure=1 is fine and will not hurt dev_reset_default.